src/AnnotationReader.php

```diff
@@ -86,7 +87,7 @@ public function __construct(private Reader $reader, private string $mode = self:
      *
      * @template T of object
      */
-    private function getClassAnnotation(ReflectionClass $refClass, string $annotationClass): ?object
```

Comment:
What's wrong with using the nullable, `?`

Reply:
There are a ton of code changes from using the nullable typing to explicit `null` in a union type definition.  I know PHP 8.0 supports union types.  But, what's the value of this?

Reply:
`doctrine/coding-standard` forces using union types for this since https://github.com/doctrine/coding-standard/pull/266

Reply:
> There are a ton of code changes from using the nullable typing to explicit `null` in a union type definition. I know PHP 8.0 supports union types. But, what's the value of this?

This changes belongs to commit named `run cs-fix script`. Every line of code was automatically refactored by code fixer configured with [doctrine/coding-standard 10.0](https://github.com/doctrine/coding-standard)

Reply:
I guess we can roll with it - sticking with Doctrine's opinions on CS.  Personally, I don't see the appeal.  It's uglier, more confusing to figure out what's being returned from a function (like the main type being expected), and encourages the use of additional return types where a union is already being returned (why not tack on another `|array`, already have `null|Object`).

I'd love to hear some justifications for this decision.  I'm sure it's something along the lines of consistency, being explicit and more obvious that null is returned, where `?` can be overlooked.